cmd/ssl-expiry-check/main.go

```diff
@@ -64,11 +115,13 @@ func main() {
 	reportErr := reportKHSuccess()
 	if reportErr != nil {
 		log.Error(reportErr)
+
 	}
+	os.Exit(0)
 }
 
+// runExpiry runs the SSL expiry check from the ssl_util package with the specified env variables
```

Comment:
in this func:
`if certExpired == true` --> `if certExpired`
`if expirePending == true` --> `if expirePending`

Reply:
Updated.